bpf/tests/scapy/parser.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+
+# Copyright Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import re
+import sys
+import jinja2
+
+from scapy.all import *
+from pkt_defs import *
+
+# Match SCAPY_BUF(name, ...
+PKT_REGEX = re.compile(
+    r'SCAPY_DEF_BUF\(\s*([A-Za-z0-9_]+)\s*,\s*(.+?)\s*$',
```

Comment:
This might cause some awkward cases, like if someone writes a comment regarding some usage of `SCAPY_DEF_BUF(...)` that might accidentally get picked up here and expanded.

Reply:
Good catch,

I will start the regex with `^'\s*` to match newline + any space, tab etc to avoid this. With the style of comments that we support (`checkpatch.pl`), that should be sufficient.